src/db/schema.js

```diff
@@ -25,6 +25,29 @@ module.exports = (db) => {
             GROUP BY 
                 short_id;
                 
+        CREATE TABLE IF NOT EXISTS sample_state (
+            sample_state_id INTEGER PRIMARY KEY AUTOINCREMENT,
+            session_short_id TEXT NOT NULL,   
+            session_sample_id INTEGER NOT NULL,     
+            version INTEGER NOT NULL DEFAULT 0,
+            content TEXT NOT NULL,
+            summary TEXT,
+            annotation TEXT,
+            created_at TEXT NOT NULL DEFAULT CURRENT_TIMESTAMP
+        );
+        
+        CREATE VIEW IF NOT EXISTS latest_sample_state AS
+            SELECT 
+                *
+            FROM 
+                (
+                  select *
+                        from sample_state
+                        order by sample_state_id DESC
+                  ) as sample_state
+            GROUP BY 
+                session_short_id, session_sample_id;
```

Comment:
this table isn't needed because historical sample_states aren't kept

Reply:
*view

Reply:
done